tests/test_cli.py

```diff
@@ -20,28 +20,28 @@ def cd_tmp_path(tmp_path: Path, monkeypatch: MonkeyPatch):
 
 
 def test_pmg_analyze(cd_tmp_path: Path):
-    exit_status = os.system(f"pmg analyze {TEST_FILES_DIR}/io/vasp/fixtures/scan_relaxation")
+    exit_status = os.system(f"pmg analyze {TEST_FILES_DIR}/io/vasp/fixtures/scan_relaxation")  # noqa:S605
```

Comment:
maybe the right time to switch to `subprocess.run` here over `os.system`? don't have a strong opinion either way

Reply:
Okay, I would prefer the flexibility from `subprocess` too. Also noticed a violation of [S602](https://docs.astral.sh/ruff/rules/subprocess-popen-with-shell-equals-true/) in `io.packmol` too, I would fix that one as it looks quite dangerous.

Reply:
Done in fdf52f16f707fbc0ec782b1af9b8c57d927188f0, just did a global search and these look like the only usage of `os.system` across the code base.